src/commons/bgp/analyser.rs

```diff
@@ -84,8 +84,8 @@ impl BgpAnalyser {
                 .collect(),
         };
 
-        let (roas, roas_not_held): (Vec<RoaDefinition>, _) = roas
-            .iter()
+        let (roas_held, roas_not_held): (Vec<ConfiguredRoa>, _) = roas
+            .into_iter()
             .partition(|roa| resources_held.contains_roa_address(&roa.as_roa_ip_address()));
```

Comment:
Why wasn't this call to `contains_roa_address()` needed before?

Reply:
that line is actually unchanged.

Reply:
Ignore me ... sigh.